### PR TITLE
Grzegorzsn/small refactors

### PR DESCRIPTION
**Pull request by grzegorzsn**
Hello,

I have prepared some small improvements for the repo. Please take a look :)

### CODE REVIEW

File: playbooks/slack.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+- name: Slack
+  hosts: 127.0.0.1
+  connection: local
+  tasks:
+    - name: Check if slack is installed
```
Comment (wyhasany) [inline playbooks/slack.yml]: You can do it in one task, like below:
```
  - name: Add NordVPN Repository from deb package from Internet
    apt:
      deb: https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/nordvpn-release_1.0.0_all.deb
    tags:
    - nord-vpn
    become: true
```

See more here: https://docs.ansible.com/ansible/latest/modules/apt_module.html

**Review by wyhasany — COMMENTED** [review]
First of all thank you for your contribution. Please let's see my code review remarks.

**Comment by grzegorzsn** [comment]
Thanks for review, updated.

**Comment by wyhasany** [comment]
That's looks much cleaner! Thank you so much!

**Pull request merged**